ama-cli/src/main/kotlin/org/apache/amaterasu/ama/cli/container/ContainerHandler.kt

```diff
@@ -0,0 +1,67 @@
+package org.apache.amaterasu.ama.cli.container
+
+import com.github.dockerjava.api.DockerClient
+import com.github.dockerjava.api.model.BuildResponseItem
+import com.github.dockerjava.core.DefaultDockerClientConfig
+import com.github.dockerjava.core.DockerClientBuilder
+import com.github.dockerjava.core.command.BuildImageResultCallback
+import com.github.dockerjava.jaxrs.JerseyDockerCmdExecFactory
+import org.slf4j.LoggerFactory
+import java.io.File
+
+
+class ContainerHandler(val action : String) {
+  val logger = LoggerFactory.getLogger(ContainerHandler::class.java)
```

Comment:
You can simply derive from KLogging

Reply:
i know, i was just trying to make it work, at this phase, will switch to klogging